osmium-builder.cpp

```diff
@@ -54,11 +54,11 @@ osmium_builder_t::get_wkb_node(osmium::Location const &loc) const
 }
 
 osmium_builder_t::wkbs_t
-osmium_builder_t::get_wkb_line(osmium::WayNodeList const &nodes, bool do_split)
+osmium_builder_t::get_wkb_line(osmium::WayNodeList const &nodes, double split_at)
 {
     wkbs_t ret;
 
-    double const split_at = m_proj->target_latlon() ? 1 : 100 * 1000;
+    bool do_split = split_at > 0.0;
```

Comment:
I would expect that comparison against zero is reasonably safe. I was considering using NaN but it did feel like a misusing the value. The other option is to add do_split as an additional parameter, which would be fine to me as well.